pkg/minikube/machine/start.go

```diff
@@ -157,7 +157,7 @@ func createHost(api libmachine.API, cfg *config.ClusterConfig, n *config.Node) (
 	cstart := time.Now()
 	glog.Infof("libmachine.API.Create for %q (driver=%q)", cfg.Name, cfg.Driver)
 
-	if err := timedCreateHost(h, api, 4*time.Minute); err != nil {
+	if err := timedCreateHost(h, api, cfg.StartHostTimeout); err != nil {
```

Comment:
We want to be backwards compatible here, so if StartHostTimeout isn't set in cfg for some reason, we still want to default back to 4 minutes.

Reply:
I am ok with default being changed, I think we could and should re-use the existing flag we have
but update the description

```
startCmd.Flags().Duration(waitTimeout, 6*time.Minute, "max time to wait for kubernetes or host to be healthy")
```

Reply:
I agree with the reuse of the ``waitTimeout`` flag as @medyagh suggests. Do we default to 6 mins instead of the current 4 if the user doesn't specify any timeout  (to maintain consistency)?

Reply:
Yes